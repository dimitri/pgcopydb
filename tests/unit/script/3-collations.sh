#! /bin/bash

set -x
set -e

# This script expects the following environment variables to be set:
#
#  - PGCOPYDB_SOURCE_PGURI

pgcopydb list collations -q --dir /tmp/collations 2>&1
