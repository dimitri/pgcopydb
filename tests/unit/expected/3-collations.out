       OID |                 Name |          Object name
-----------+----------------------+---------------------
     12329 |           en_US.utf8 |    database postgres
     12440 |          de-DE-x-icu | column f2 of table colls 
     12559 |          en-US-x-icu | column f2 of table mycoltab 
     12658 |          fr-FR-x-icu | column f1 of table colls 
     16418 |                mycol | column f1 of table mycoltab
